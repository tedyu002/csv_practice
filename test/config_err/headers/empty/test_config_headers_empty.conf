CSV_INPUT=test.input
CSV_OUTPUT=test.out
CSV_ERROR=test.err
CSV_RESULT=test.res
