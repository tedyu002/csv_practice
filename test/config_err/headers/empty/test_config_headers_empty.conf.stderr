No HEADERS
config 'test_config_headers_empty.conf' parse error
